Tests/test_MafIO_index.py

```diff
@@ -315,8 +315,12 @@ def test_records_end(self):
 
             fetched_recs = self.idx._get_record(99228)
 
-            for i in range(6):
-                self.assertTrue(compare_record(recs[i], fetched_recs[i]))
+            self.compare_record(recs[0], fetched_recs[0])
+            self.compare_record(recs[1], fetched_recs[1])
+            self.compare_record(recs[2], fetched_recs[2])
+            self.compare_record(recs[3], fetched_recs[3])
+            self.compare_record(recs[4], fetched_recs[4])
+            self.compare_record(recs[5], fetched_recs[5])
```

Comment:
Do you find the unrolled loop clearer? To be it is a recipe for human error in the indices.

Reply:
I am fine either way, but with looping if there is an error or failure, you won't know which of the 6 records caused it.

Reply:
That is a good argument. Thanks.